chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -590,7 +597,15 @@ where we can write any vector in our space
 as a weighted sum of these *basis vectors*.
 -->
 
-*dịch đoạn phía trên*
+<> Tuy nhiên, điều này cho ta thấy cách một ma trận biến đổi *bất kỳ* vector nào
+thông qua cách nó biến đổi *hai vector cụ thể*:
+$[1,0]^\top$ và $[0,1]^\top$.
+Quan sát một chút, chúng ta thực tế đã thu gọn một bài toán vô hạn
+(tính toán cho bất kỳ vector nào) thành một bài toán hữu hạn
+(tính toán cho chỉ hai vector). Những vector này là ví dụ của *cơ sở*,
```

Comment:
(tính toán cho chỉ hai vector). Hai vectors này còn có tên gọi khác là *vector cơ sở* - có nghĩa là thông qua chúng, ta có thể biểu diễn bất kì vector nào trong không gian dưới dạng tổng có trọng số ...